bundles/org.openhab.binding.monopriceaudio/src/main/java/org/openhab/binding/monopriceaudio/internal/handler/MonopriceAudioHandler.java

```diff
@@ -0,0 +1,723 @@
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EventObject;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.OpenClosedType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.StateOption;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.eclipse.smarthome.io.transport.serial.SerialPortManager;
+import org.openhab.binding.monopriceaudio.internal.MonopriceAudioException;
+import org.openhab.binding.monopriceaudio.internal.MonopriceAudioStateDescriptionOptionProvider;
+import org.openhab.binding.monopriceaudio.internal.communication.MonopriceAudioCommand;
+import org.openhab.binding.monopriceaudio.internal.communication.MonopriceAudioConnector;
+import org.openhab.binding.monopriceaudio.internal.communication.MonopriceAudioDefaultConnector;
+import org.openhab.binding.monopriceaudio.internal.communication.MonopriceAudioIpConnector;
+import org.openhab.binding.monopriceaudio.internal.communication.MonopriceAudioMessageEvent;
+import org.openhab.binding.monopriceaudio.internal.communication.MonopriceAudioMessageEventListener;
+import org.openhab.binding.monopriceaudio.internal.communication.MonopriceAudioSerialConnector;
+import org.openhab.binding.monopriceaudio.internal.communication.MonopriceAudioZone;
+import org.openhab.binding.monopriceaudio.internal.configuration.MonopriceAudioThingConfiguration;
+import org.openhab.binding.monopriceaudio.internal.dto.MonopriceAudioZoneDTO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link MonopriceAudioHandler} is responsible for handling commands, which are sent to one of the channels.
+ *
+ * Based on the Rotel binding by Laurent Garnier
+ *
+ * @author Michael Lobstein - Initial contribution
+ */
+@NonNullByDefault
+public class MonopriceAudioHandler extends BaseThingHandler implements MonopriceAudioMessageEventListener {
+    private static final long RECON_POLLING_INTERVAL = TimeUnit.SECONDS.toSeconds(60);
```

Comment:
The `TimeUnit.SECONDS.toSeconds` doesn't do anything here so you can omit it. If you are trying to make your assignment more declarative then I suggest you add a `_SEC` suffix to your constant name instead.
```suggestion
    private static final long RECON_POLLING_INTERVAL_SEC = 60;
```